### PR TITLE
Android: Granting execute permission for gradlew

### PR DESCRIPTION
**Pull request by yashovardhan99**
In [my android repo](https://github.com/yashovardhan99/HealersDiary), while testing this action, I noticed `./gradlew build` fails because it does not have execute permissions. Adding the execute permissions using `chmod` fixes this issue. Refer to [this action log](https://github.com/yashovardhan99/HealersDiary/commit/193206639f00b7850292c1c6f2184492ec96b16a/checks).

### CODE REVIEW

**Comment by joshmgross** [comment]
👋  Thanks for your contribution and apologies for the delay. This change looks good to me, could you resolve the conflicts?

**Comment by yashovardhan99** [comment]
Wow, it's been quite a while.. I will have a look at it tomorrow

**Comment by yashovardhan99** [comment]
Hey @joshmgross, I have tried resolving the conflicts :face_with_head_bandage:. Please check if it's fine now?

**Comment by yashovardhan99** [comment]
Hey @joshmgross, I have tried resolving the conflicts :face_with_head_bandage:. Please check if it's fine now?

File: ci/android.yml [hunk 1]
```diff
@@ -1,21 +1,23 @@
-name: Android CI
```
Comment (joshmgross) [inline ci/android.yml:1]: Could you double-check the line endings here?

It looks like this is reverting the change from https://github.com/actions/starter-workflows/pull/428/files

Comment (yashovardhan99) [reply ci/android.yml:1]: Should be fixed now

Comment (joshmgross) [reply ci/android.yml:1]: I'm still seeing a large PR diff. 

Could you run `git add --renormalize ci/android.yml` and commit those changes?

Comment (yashovardhan99) [reply ci/android.yml:1]: Alright give me a few minutes

**Review by joshmgross — APPROVED** [review]
Thank you!

File: ci/android.yml (hunk 1 above)
Comment (yashovardhan99) [reply ci/android.yml]: It seems good now. My fault using the github editor to resolve conflicts earlier. That messed up the line endings.

**Pull request merged**